Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -646,7 +655,7 @@ open class ComposerVC: _ViewController,
             )
         } else {
             usersCache = searchUsers(
-                channel.watchers.map { $0 } + channel.cachedMembers.map { $0 },
+                channel.watchers.map { $0 } + (memberListController?.members.compactMap { $0 } ?? []),
```

Comment:
`memberListController?.members.compactMap`

won't this impact performance as well? 
iterating on long arrays is N complexity

Reply:
No, it won't, it's maxed at 100 members. Either way, we should not prematurely optimize for performance, but for readability instead.